### DISCUSSION TITLE
How to get more people to register on SwitchScores

### DISCUSSION THREAD

**Discussion by cosmo0** [benbarden/switchscores#46] [Ideas]
In order to get users to register and create a community, several things could be improved.

**Incentives to register**: add messages like "register to do X" in as many places as possible. Register to add this game to your collection, register to add a quick review, register to help with the database...

**Make the site more useable for international users**: for instance, currently, the prices are displayed only in £, which is only useful for UK users. I think displaying the price in € and $ (and US sales) would help a bit, although not a lot. Displaying at least £ and € would be a huge improvement: € and $ prices are often the same (a 10€ game is 10$ in the US), but I have no idea how much a £ is worth.

**Create unique features**; I really think my idea of a browser extension that inserts infos directly in the page (see #19) would be a game-changer for many people: a simple way to see if this "-90% game" is worth it. Although it's probably not easy, especially since ideally it would require to match the games to all EU stores and the US one, but I'm convinced it would be worth it.

**Just talk more about the site**. Get people to talk about it on Twitter, Facebook, Reddit, Resetera, whatever. It's also one of the more time consuming and less rewarding ways, especially since you don't gain money from visitors, but following a few forums and posting a useful link to the site once in a while might help.

**Better SEO**: (issue: #40 ), meta description tags, descriptions in categories ("Adventure games are..."): content is king in SEO, more relevant content = better SEO. Better titles: titles that match what people google, like "best X Switch games" as category list titles

**Comment by cosmo0** [comment]
@benbarden FYI I think renaming "category" to "genre" (to help user know what it's about), and changing the titles of categories to "Best [genre] games on Nintendo Switch" would help a lot for SEO. I have a few "best games" lists on my site, and they're the pages that get the most views despite me not linking to them ever.

**Reply by benbarden** [reply]
Aw, it doesn't auto-link commits if I link to a discussion! It works with issues.

I'll manually link the commit here - https://github.com/benbarden/switchscores/commit/4b609d1b645a6c75f87d6215eb92ccbe02db2170

**Comment by benbarden** [comment]
In this week's Review highlights post, I've reinstated the Unranked list and added a link to encourage people to register. A small thing but it might help.
https://www.switchscores.com/news/2021-01-30/switch-review-highlights-30th-january-2021

**Reply by cosmo0** [reply]
Cool! But you display "TBC" instead of the number of reviews, there?
